deploy/tests/test_utils.py

```diff
@@ -1,37 +1,50 @@
 from pathlib import Path
 from unittest.mock import MagicMock, call
-from util.fsspec_utils import downloadURI
+from util.io import downloadURI
```

Comment:
Looking at the moto docs you sent me, I'm reminded that they recommend that you defer importing code that can reach out to AWS until after you have your mocks setup (and  you have the fake credentials installed). see http://docs.getmoto.org/en/latest/docs/getting_started.html#what-about-those-pesky-imports

I think that I'll open another PR that flattens aws_test_credentials into conftest.py. Or maybe you could after we're done with this one.

Reply:
Agreed, if a boto3 is outside of a function, it'll be executed before there's a chance to mock it.